src/app/framework-modules/elements/elements.ts

```diff
@@ -224,12 +226,14 @@ import { CtHorizontalBarChartComponent } from './ct-horizontal-barchart/componen
         CtNumberCardComponent,
         CtLinearGaugeComponent,
         CtHorizontalBarChartComponent,
+        CTFieldGen,
         /**
          * Miscellaneous Exports
          */
         ReactiveFormsModule,
         CommonModule,
         FormsModule,
+        
     ]
 })
   export class ElementsModule {
```

Comment:
I thought you had a linter installed?  If you do and your linter catches these things, but you forgot to run this new code through the linter, then I would highly recommend installing a linter as part of your PR checks.  Your linter could do one of the following:
Option 1. The linter lints and reformats the code up on the CI server and recommits it somehow. This is the more aggressive option.
Option 2. The linter lints (up on the CI server) and if the git repo is dirty after that (that is, the linter has made any changes), then your PR check fails. This is the more common conservative option.